ngraph/core/include/ngraph/op/group_conv.hpp

```diff
@@ -96,7 +96,7 @@ namespace ngraph
             NGRAPH_SUPPRESS_DEPRECATED_START
```

Comment:
Please remove `NGRAPH_SUPPRESS_DEPRECATED_START` and `NGRAPH_SUPPRESS_DEPRECATED_END` macros